resources/pre-release-week.md

```diff
@@ -46,6 +46,10 @@ The script automates the following tasks:
 
 ## 💻 Coding
 
+⚠️ [`meilisearch-js`](https://github.com/meilisearch/meilisearch-js) and [`instant-meilisearch`](https://github.com/meilisearch/instant-meilisearch/) implementations should start as soon as possible since:
+- they are related to the [`mini-dashboard`](https://github.com/meilisearch/mini-dashboard).
```

Comment:
```suggestion
- they are dependencies of the [`mini-dashboard`](https://github.com/meilisearch/mini-dashboard).
```

Should we mention: 
- docs-searchbar
- gatsby
- firebase
- Demos (devrel and playgrounds)
- Strapi
- (Saas in the future?)

?

Reply:
We should not list the dependencies here. We are in the pre-release context, that's why the mini-dashboard is mentioned.

Reply:
I plan to do a guide gathering all the dependencies in integration team (Laura asked for it anyway)